website/docs/misc/i18next.md

```diff
@@ -192,7 +200,19 @@ i18next.t("key", { count: 5 }); // -> "items"
 
 ## Context
 
-By providing a context you can differ translations. Both i18next and Lingui have the context feature to differentiate messages.
+By providing a context you can differ translations for the same sentences, or give more details to translators. Both i18next and Lingui have the context feature to differentiate messages. However, Lingui offers an automatic additional "context" by including in the `.po` file the locations where each message is used - i18next can't do that from their plain JSON files.
+
+i18next sample:
+
+```js
+i18next.t("Hello!", { context: "This goes in the header" });
+```
+
+Lingui sample:
+
+```js
+i18n._("Hello!", { comment: "This goes in the header" });
+```
```

Comment:
Unfortunately that will not work. Most lingui features build around macros, pieces of code which executed during build step not a runtime. This helps reduce size footprint of the translation catalogs as well as library itself.

In this case to specify a context for a message you MUST use a macro like this

```js
import {t} from "@lingui/macro";

t({message: `From: `, context: "direction"})
```

With plain runtime modules (i18n._) the only way is custom id.